mods/ra/rules/husks.yaml

```diff
@@ -1,16 +1,3 @@
-1TNK.Husk:
```

Comment:
Are we sure that this isn't used in any maps?

Reply:
Wait, why do we remove the husk entirely from code instead of just removing
https://github.com/OpenRA/OpenRA/blob/4df5ac03850213a8002ee3171c3eb2c6154e3931/mods/ra/rules/vehicles.yaml#L89-L90

Reply:
I can keep the husk, but figured it is basically dead code if we don't use it.

Reply:
We should remove it if we're sure its not used.

Reply:
https://github.com/OpenRA/OpenRA/search?q=1TNK.Husk&unscoped_q=1TNK.Husk shows that it is only used here.